README.md

```diff
@@ -1,7 +1,16 @@
 # TFExplorer
+[![Build Status](https://travis-ci.org/hardlyhuman/TFExplorer.svg?branch=master)](https://travis-ci.org/hardlyhuman/TFExplorer)
+![alt text](https://patrolavia.github.io/telegram-badge/chat.png "Logo title")
+**TFExplorer** enables developers to quickly find out the relevant TensorFlow API's for their needs & requirements.
```

Comment:
Please add line break here. Putting a double space and Enter should give space

Reply:
SImply using "ENter" does not break a line in markdown. You should use 2 space at the line end and then ENter to break